session/session.go

```diff
@@ -375,7 +375,7 @@ func (s *session) doCommitWithRetry(ctx context.Context) error {
 			// For autocommit single statement transactions, the history count is always 1.
 			// For explicit transactions, the statement count is more than 1.
 			history := GetHistory(s)
-			if history.Count() > 1 {
+			if history.Count() > 1 && strings.Contains(err.Error(), util.WriteConflictMarker) {
```

Comment:
can this be replaced with `strings.HasPrefix`?

Reply:
and I have another irrelevant question~

If we have  a tx with two statements, select then insert... maybe we should not  retry insert stmt too? event if `history.count() == 1`

Reply:
Yes, this condition is not correct.

Reply:
I've tested it. It is correct...